roadmap.md

```diff
@@ -23,26 +23,40 @@ implement the spec.
 1. Create and deploy a website that features a simple overview, email list and directs visitors to Github.
 1. Store all website assets in the CloudEvents repository, under the governance
 of the working group.
-
-*0.2*
-
 1. Have at least 2 implementations of the specification that can demonstrate interoperability.
 1. Include a specification for mapping the CloudEvents specification to HTTP.
 1. Include a specification for mapping the CloudEvents specification to JSON.
 1. Changes to the spec to facilitate adoption.
-1. Draft documentation and developer guide.
 1. Publicize at conferences ([CloudNativeCon Europe](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/)).
-
-*0.3*
-
 1. Interoperability demo.
     1. At least one open source implementation of sending and receiving events.
     1. Events are sent by code written by Developer1 and recieved by code written by Developer2, where Developer1 has no knowledge of Developer2.
     1. Draft trigger specification (e.g. how events + actions are associated).
```

Comment:
Move to 0.2 -- did I miss this? if so, pls add link... In any case, I believe I was tracking all changes up to 0.1 so would be part of next milestone, it's outside of the 0.1 specification, so leaving it here is a bit confusing.

Reply:
hmm, since in the original roadmap this was listed under the interop demo line I assumed it was all just related to that - and not suggesting a new spec. I'll remove it since it will just lead to confusion.